sklearn/datasets/_samples_generator.py

```diff
@@ -169,13 +173,31 @@ def make_classification(
         for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    return_X_y : bool, default=True
+        If True, the ``(data, target)`` instead of a Bunch object.
+
     Returns
     -------
-    X : ndarray of shape (n_samples, n_features)
-        The generated samples.
-
-    y : ndarray of shape (n_samples,)
-        The integer labels for class membership of each sample.
+    data : :class:`~sklearn.utils.Bunch`
+        Dictionary-like object, with the following attributes.
+
+        DESCR : str
+            A description of the function that generated the dataset.
+        generator_parameter : dict
+            A dictionary that stores the values of the arguments passed to the
+            generator function.
+        feature_info : list of len(n_features)
+            A description for each generated feature. "US" for useless, "IN" for
+            informative, "RP" for repeated and "RD" for redundant.
```

Comment:
I think I prefer `{"random", "informative", "repeated", "redundant"}`. No need to have a two letter abbreviation here.